terraform/alb.tf

```diff
@@ -65,9 +65,9 @@ resource "aws_alb_target_group" "service" {
     interval            = "30"
     protocol            = "HTTP"
     matcher             = "200"
-    timeout             = "3"
+    timeout             = "8"
```

Comment:
Because our Service is a bit slow to start up 🐌 we found that the Service tasks were often labelled as `unhealthy` by the load balancer, which would then kill it and bring up a new task, when they were definitely okay!

So we increased the timeout (and unhealthy threshold) to prevent this from happening.